src/js/view/controls/controls.js

```diff
@@ -361,6 +361,7 @@ export default class Controls {
 
     disable(model) {
         const { nextUpToolTip, settingsMenu, infoOverlay } = this;
+        const volumeTooltip = this.controlbar.elements.volumeTooltip;
```

Comment:
this is always undefined because in `controlbar.js`, volumetooltip is not camel cased. Let's make this `volumetooltip` for now.
 I suggest we call destroy on the controlbar instead of calling it directly on the volumetooltip. THe controls should not know about the internals of the controlbar, it should just interact with its interface. in other words:
```
const { nextUpToolTip, settingsMenu, infoOverlay,  controlbar } = this;
.
.
.
if (controlbar) {
         controlbar.destroy();
}
```
and then add a destroy function in controlbar that calls destroy on the all lower case volumetooltip

Reply:
But disabling doesn't destroy the controlbar?